### ISSUE TITLE
TTF Artifacts not displayed

### ISSUE THREAD

**Issue by FINCONS-IBD** [ngdenterprise/visual-token-designer#23]
I've installed the last version of the Visual Token Designer; In Visula Studio Code I see the "tau" icon and the 3 views on its right when I select the Token Designer icon, but the 3 views are empty and don't react if I select the "refresh" icon on these views.
My PC has an Intel 64 bits CPU and Windows 10 20H2.

Do I missed some step?
Thanks in advance for any hint

**Comment by magick93** [comment]
Same problem. 

I think there is a grpc npm module that needs to be updated. 

But it seems this project has been abandoned.

**Comment by mwklein** [comment]
I have the same issue. I even tried to build the grpc module in node v14 but couldn't get it to perfectly match the version of node VSCode was compiled. @marleyg or @JohndeVadoss is it possible to rebuild the extension for multiple versions of VSCode?

**Comment by JohndeVadoss** [comment]
Yes @mwklein working on this now - it is time to update this and sync

**Comment by JohndeVadoss** [comment]
quick update - building the current release and running on Visual Studio Code v1.53 works fine - as a temporary work around if someone needs to demo. I am working on the gRPC compat issues and will have a new release in a few days. @mwklein @marleyg
